tests/Doctrine/Tests/ORM/Mapping/MappingDriverTestCase.php

```diff
@@ -504,23 +504,23 @@ public function testDefaultFieldType(): void
         self::assertArrayHasKey('id', $class->fieldMappings);
         self::assertArrayHasKey('name', $class->fieldMappings);
 
-        self::assertArrayHasKey('type', $class->fieldMappings['id']);
-        self::assertArrayHasKey('type', $class->fieldMappings['name']);
+        self::assertNotNull($class->fieldMappings['id']->type);
+        self::assertNotNull($class->fieldMappings['name']->type);
```

Comment:
Do we need those at this point? It's not an array anymore and if `type` isn't initialized, we would get a "must not be accessed before initialization" (but afaik the props are initialized by default). Nevertheless, the actual value is checked in the next lines. Same with the other assertNotNull methods below.